src/devices/vdml_usd.cpp

```diff
@@ -21,5 +21,80 @@ std::int32_t is_installed(void) {
 	return usd_is_installed();
 }
 
+std::int32_t list_files_raw(const char* path, char* buffer, int32_t len) {
```

Comment:
std::string_view

Reply:
> std::string_view

I understand that's more C++onic but for the sake of our users I think a std::string would be a little more approachable... saves everyone a google search.

Reply:
😖 wasted allocation for every call with a string literal for path , which I assume is the typical case.

Reply:
> make sure all of the std::strings are also references pls*
If you choose to use `std::string` instead of `string_view` do make sure its `const` refs, as mutable refs will cause other issues (won't accept temporaries i.e. string literals that convert to `std::string`)